src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_7001_8000.cpp

```diff
@@ -585,6 +586,55 @@ GUI_TEST_CLASS_DEFINITION(test_7293) {
     GTMenu::clickMainMenuItem(os, {"File", "Open as..."});
 }
 
+GUI_TEST_CLASS_DEFINITION(test_7367) {
+    // Generate a large sequence.
+    // Check that test does not time-outs and the generated sequence contains expected base distribution.
+
+    DNASequenceGeneratorDialogFillerModel model(sandBoxDir + "/test_7370.fa");
+    model.percentA = 10;
+    model.percentC = 20;
+    model.percentG = 30;
+    model.percentT = 40;
+    model.length = 100 * 1000 * 1000;
```

Comment:
`100 * 1024 * 1024`?

Reply:
This one can be any big number that will make the test slow or will make the . It has not relation to low-level powers of 2.